book/from_flat_php_to_symfony2.rst

```diff
@@ -98,11 +102,12 @@ the code that prepares the HTML "presentation"::
     require 'templates/list.php';
 
 
-The HTML code is now stored in a separate file (``templates/list.php``), which
+The HTML code is now stored in a separate file ``templates/list.php``, which
```

Comment:
Changing the order to "in a separate `templates/list.php` file" seems to be better (although I'm not a native).